destinations/data/seq.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: internal.odigos.io/v1beta1
+kind: Destination
+metadata:
+  type: seq
+  displayName: Seq
+  category: self-hosted
+spec:
+  image: seq.svg
+  signals:
+    traces:
+      supported: true
+    metrics:
+      supported: false
+    logs:
+      supported: true
+  fields:
+    - name: SEQ_ENDPOINT
+      displayName: OTLP HTTP Endpoint
+      componentType: input
+      componentProps:
+        required: true
+        tooltip: 'The format is `host:port`, `host` is required and equals the cluster internal DNS record of the Seq instance, `port` is required and equals any of the exposed ports of the Seq instance (defaults to 5341), do not add the rest of the path `/ingest/otlp/v1/*`.'
```

Comment:
`host:port` is how gRPC endpoint usually looks like.

For http endpoint, it's a convention to also include the scheme (`http` / `https`).